workloads/network-perf/smoke_test.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+set -x
+
+source ./common.sh
+
+if [[ ${COMPARE} == "true" ]]; then
+  _baseline_pod_1p_uuid=
+  _baseline_pod_2p_uuid=
+  _baseline_pod_4p_uuid=
+fi
+
+if [[ ${BASELINE_POD_1P_UUID} ]]; then
+  _baseline_pod_1p_uuid=${BASELINE_POD_1P_UUID}
+fi
+
+if [[ ${BASELINE_POD_2P_UUID} ]]; then
+  _baseline_pod_2p_uuid=${BASELINE_POD_2P_UUID}
+fi
+
+if [[ ${BASELINE_POD_4P_UUID} ]]; then
+  _baseline_pod_4p_uuid=${BASELINE_POD_4P_UUID}
+fi
+
+oc -n my-ripsaw delete benchmark/uperf-benchmark
+
+for pairs in "${client_server_pairs[@]}"
+do
+
+cat << EOF | oc create -f -
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: uperf-benchmark
+  namespace: my-ripsaw
+spec:
+  elasticsearch:
+    server: $_es
+    port: $_es_port
+  clustername: $cloud_name
+  test_user: ${cloud_name}-smoke-${pairs}p
+  metadata:
+    collection: ${_metadata_collection}
+    serviceaccount: backpack-view
+    privileged: true
+  cerberus_url: "$CERBERUS_URL" 
+  workload:
+    name: uperf
+    args:
+      hostnetwork: false
+      serviceip: false
+      pin: $pin
+      pin_server: "$server"
+      pin_client: "$client"
+      multus:
+        enabled: false
+      samples: 1
+      pair: ${pairs}
+      nthrs:
+        - 1
+      protos:
+        - tcp
+      test_types:
+        - stream
+      sizes:
+        - 1024
+      runtime: 30
+EOF
+
+sleep 30
```

Comment:
What's the need for this sleep?

Reply:
it takes some time for the benchmark operator to come up